src/Patterns/templates/Scd2Synapse.twig

```diff
@@ -1 +1,94 @@
--- TODO --
```

Comment:
Skopiroval som SNFK sablonu do Synapse, aby bolo lepsie vidiet rozdiel, ked to upravis.